ui/pages/confirmations/components/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -59,14 +72,34 @@ function BlockaidBannerAlert({ txData, ...props }) {
 
   const t = useContext(I18nContext);
   const { updateTransactionEventFragment } = useTransactionEventFragment();
+  const [isLoadingDisplayed, setIsLoadingDisplayed] = useState(false);
+
+  useEffect(() => {
+    if (securityAlertResponse?.reason === 'loading') {
```

Comment:
looks like we have Blockaid reason and result type loading values mixed up in our repo (prior to this PR). Your condition appears to be correct here based on existing code and app/scripts/lib/transaction/util.test.ts. 

Possibly the values are swapped in shared/constants/security-provider.ts? 

<img width="526" alt="Screenshot 2024-02-20 at 12 25 43 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/a5b5fdb3-893c-49ab-b1c1-698c45be03a1">
<img width="1471" alt="Screenshot 2024-02-20 at 12 26 25 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/9f9ff61f-44fb-4dbd-8eaa-5abae0cbb034">

---
After fixing the mixup, we should use the appropriate constant variable here

Reply:
This mixing is happening in controller layer, in UI layer I am only comparing the values. We can create a separate task to fix this.